docs/pages/choose-an-edition/teleport-cloud/faq.mdx

```diff
@@ -25,9 +25,9 @@ Yes. Customers can subscribe to Teleport Enterprise Cloud updates at [status.tel
 
 Please [reach out to sales](https://goteleport.com/signup/enterprise) to discuss pricing.
 
-## If I start with Teleport Enterprise Cloud, can I move to Teleport Enterprise or Teleport Open Source, or do I need to start again?
+## If I start with Teleport Enterprise Cloud, can I move to Teleport Enterprise or Teleport Open Source?
 
-If you plan to use S3 and DynamoDB as storage backends, we can provide data for you to import. But you should reach out to us first. If you use a different backend, you will need to start over.
+If you plan to use S3 and DynamoDB as storage backends, we can provide data for you to import. If you use a different backend, you can reuse configurations in a new Teleport cluster. Prior to any planned migration we recommend contacting Teleport to get the best options.
```

Comment:
This isn't clear to me.

By _reuse configuration_ do you mean you can use the same configuration but will have to manually migrate the data?

Reply:
Yes, some of the configurations can be reused directly (roles) and others will require some changes.  That will require a new cluster but does not require starting completely over. Starting completely over sounds much starker then the reality.

Reply:
I think you need to find a way to add that to the PR. As-is, this PR leaves me with more questions than answers.

Reply:
@stevenGravy Does "configurations" mean "Teleport configuration resources" here?